### ISSUE TITLE
 Video 'xxxxxxxxxxx' is unplayable. Reason: 'Please sign in'

### ISSUE THREAD

**Issue by atozfugh** [taylorbobaylor/TrailerDownloader#56]
Hi @taylorbobaylor, 

first of all, nice app!

I started the windows version and can successfully see all my movies in there. When i try to download any trailer i get the error, that i need so sign in. I'm pretty sure this is some kind of ytdl error, but im not quite sure how to actually successfully login.

Thanks in advance! :) 

```
info: Microsoft.Hosting.Lifetime[14]
      Now listening on: http://localhost:5000
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
      Content root path: C:\Users\ulti\Downloads\TrailerDownloaderWindows
warn: Microsoft.AspNetCore.HttpsPolicy.HttpsRedirectionMiddleware[3]
      Failed to determine the https port for redirect.
fail: TrailerDownloader.SignalRHubs.MovieHub[0]
      Error downloading trailer for 12 Years a Slave
Video 'zWottOuvRyU' is unplayable. Reason: 'Please sign in'.
fail: TrailerDownloader.SignalRHubs.MovieHub[0]
      Error downloading trailer for 12 Years a Slave
Video 'zWottOuvRyU' is unplayable. Reason: 'Please sign in'.
fail: TrailerDownloader.SignalRHubs.MovieHub[0]
      Error downloading trailer for 25 km+h
Video 'tv1OhrqabR4' is unplayable. Reason: 'Please sign in'.
fail: TrailerDownloader.SignalRHubs.MovieHub[0]
      Error downloading trailer for 25 km+h
Video 'tv1OhrqabR4' is unplayable. Reason: 'Please sign in'.
fail: TrailerDownloader.SignalRHubs.MovieHub[0]
      Error downloading trailer for Bee Movie
Video 'B-XbD9U8GKA' is unplayable. Reason: 'Please sign in'.
```

**Comment by taylorbobaylor** [comment]
Hey there! I'm actually working on a fully modernized version with auto trailer download capabilities and faster backend etc. But yes, I think you're correct. I'm not really maintaining this version right now as I'm rebranding as `Trailarr` for this repo 😛 

How are you running this? I could try updating the ytdl package and that usually fixes this kind of issue.

**Comment by Starminder** [comment]
There's already an app for trailers called trailarr.  I can't get that one to work either, but you might want to check it out before you rebrand to a name already in use.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by taylorbobaylor** [comment]
Well damn..... Had no clue LOL.... Thanks for the heads up. Looks like there's multiple `Trailarr` applications....... How do none of them work LOLOLOLOLOL......

- https://github.com/nandyalu/trailarr
- https://github.com/trueChazza/trailarr (Deprecated)
- https://github.com/jsaddiction/Trailarr

**Comment by Starminder** [comment]
That first one is fairly new, maybe 2 months.  He’s trying to do both tv shows and movies and it does work for some people.  I can get it as about as close as you can get it to working, but in the end it’s not downloading anything for me.

Trailarr is a great name, I was going to try to make one a couple of years ago and…still don’t have the time or the talent.

To me, with that name, it should utilize the same framework as the other *.arrs.

Trailer Park would be a good name…

**Comment by taylorbobaylor** [comment]
Yeah good point. Seems like overkill but I _could_ fork Sonarr/Radarr and have it with the same setup etc like Lidarr and all the others did......

**Comment by Starminder** [comment]
I think utilizing that framework and giving it a familiar look and feel will go a long way toward acceptance, help, usage, expectation, and other good things.  It automatically makes it a good app.  Just my two.

Please add tv show trailers 😊

**Comment by atozfugh** [comment]
Hi @taylorbobaylor,

thanks for your reply! I'm running it like in the description for windows.

Just starting the .exe and accessing the localhost:5000 webinterface after that.

But no worries i'm stoked to see the reworked app! :)

**Comment by Starminder** [comment]
It’s actually downloading trailers for you? I’ll give that one another try this week.

Good luck with the rebranding, if I can be of help just yell.

**Comment by taylorbobaylor** [comment]
> It’s actually downloading trailers for you?

No, it's not working for me locally after updating YouTubeExplode packages. I'll take a look when I get some time. 

So, you didn't have any luck with the new one that just came out?

Haven't had time to give that a try. If that works well, might as well deprecate this project.

**Comment by Starminder** [comment]
If you get it working please let me know.  I’ve tried both Linux and Windows with no luck.  I watch more TV Shows than Movies, so having TV Show trailers readily available is most important to me.

**Comment by jamescporter** [comment]
I've got this issue too

**Comment by Classified711** [comment]
I was getting the error also but it disapeared when I logged into youtube in the browser on the windows machine I was running the app from.

**Comment by taylorbobaylor** [comment]
> I was getting the error also but it disapeared when I logged into youtube in the browser on the windows machine I was running the app from.

Interesting..... Hmmmmm..... @jamescporter and @atozfugh could you try this with your setup and see if it works?

**Comment by untotren** [comment]
any update on this? I am not getting it working at all on windows with the same errors.  Is this project abandoned?
